e2e/features/steps/feedback.py

```
@@ -0,0 +1,120 @@
+    context: Context, conversation_id: str | None
+) -> None:
+    """Send POST HTTP request with JSON payload to tested service."""
+    endpoint = "feedback"
+    base = f"http://{context.hostname}:{context.port}"
+    path = f"{context.api_prefix}/{endpoint}".replace("//", "/")
+    url = base + path
+    payload = json.loads(context.text or "{}")
+    if conversation_id is not None:
+        payload["conversation_id"] = conversation_id
+    headers = context.auth_headers if hasattr(context, "auth_headers") else {}
+    context.response = requests.post(url, headers=headers, json=payload)
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)